tests/integration/create_token.py

```diff
@@ -1,32 +1,33 @@
 from app.views.dev_mode import generate_token, create_payload
 
-PAYLOAD = {
-    'user': "mci-integration-test",
-    "period_str": "April 2016",
-    "period_id": "201604",
-    "collection_exercise_sid": "789",
-    "ru_ref": "123456789012A",
-    "ru_name": "MCI Integration Testing",
-    "ref_p_start_date": "2016-04-01",
-    "ref_p_end_date": "2016-04-30",
-    "return_by": "2016-05-06",
-    "trad_as": "Integration Tests",
-    "employment_date": "1983-06-02",
-    "variant_flags": None,
-    "exp_time": 3600  # one hour from now
+DEFAULT_PAYLOAD = {
+    'user': 'mci-integration-test',
+    'period_str': 'April 2016',
+    'period_id': '201604',
+    'collection_exercise_sid': '789',
+    'ru_ref': '123456789012A',
+    'ru_name': 'MCI Integration Testing',
+    'ref_p_start_date': '2016-04-01',
+    'ref_p_end_date': '2016-04-30',
+    'return_by': '2016-05-06',
+    'trad_as': 'Integration Tests',
+    'employment_date': '1983-06-02',
+    'variant_flags': None,
+    'region_code': 'GB-ENG',
+    'language_code': 'en',
+    'exp_time': 3600  # one hour from now
 }
 
 
-def create_token(form_type_id, eq_id, start_date=None, end_date=None, employment_date=None, region_code=None, language_code=None):
+def create_token(form_type_id, eq_id, **metadata_kwargs):
```

Comment:
Did a similar thing for jti claim branch, now that's in it's worth rebasing and resolving conflicts